src/main/java/com/marklogic/client/ext/document/DocumentWriteOperationBuilder.java

```diff
@@ -17,6 +17,10 @@
 	private ContentIdExtractor contentIdExtractor = new DefaultContentIdExtractor();
 
 	public DocumentWriteOperation build(AbstractWriteHandle content) {
+		if (content == null) {
```

Comment:
I actually think a null pointer exception is better here. If null is passed in, it may be masking a bigger problem by returning null in response. I think it's better than it throws a null pointer so that the root problem can be identified - in this case, the root problem is that a sink task can receive a null value, which means we want to handle that in the sink task, not here.

Reply:
Done.